### DISCUSSION TITLE
Transparent Overlay?

### DISCUSSION THREAD

**Discussion by Isak-Andersson** [ImageMagick/ImageMagick#5395] [Help] [answered]
Sorry if this is a really basic question but I'm really struggling with this one for some reason. I'm trying to simply put one image on top of another with a certain level of transparency. What is the simplest way to achieve this?

I'm on Windows 10 and ImageMagick version 7.1.0.

Put this with 75% transparency:
![B](https://user-images.githubusercontent.com/78758898/181926289-295422f1-df0a-4a24-a598-a75adb082098.png)
On top of this:
![A](https://user-images.githubusercontent.com/78758898/181926288-23fb00c0-bd91-4e27-8c27-fe33e23f060a.png)
To make this:
![C](https://user-images.githubusercontent.com/78758898/181927015-09932861-9f7e-4a44-9e58-bd149d1534b1.png)

**Comment by fmw42** [comment, marked as answer]
Here are two ways. The first just dissolves the two images with the proper ratio.  The second sets the alpha channel to the proper opacity and then composites.

`magick background.png overlay.png -define compose:args=25,100 -compose dissolve -composite result1.png
`
![result1](https://user-images.githubusercontent.com/13458218/181933622-28fd2b2c-df78-4dd0-b3a5-98a28971d691.png)


`magick background.png ( overlay.png -alpha set -channel a -evaluate set 25% +channel ) -compose over -composite result2.png
`
![result2](https://user-images.githubusercontent.com/13458218/181933626-79946ec4-dd18-43d5-a744-909ce58563a1.png)


See 
https://imagemagick.org/Usage/compose/#dissolve
https://imagemagick.org/Usage/basics/#parenthesis
https://imagemagick.org/Usage/windows/
